### PR TITLE
fix after model hook

### LINKED ISSUE #285: Cursor based, API without count key
Hi there! Thanks for the awesome paging component!

I'm using ember-infinity with version 1.0.0.beta-4.

And I'm building an api with no total_pages. I'm following the tutorial in README but I'm having a problem because, `_canLoadMore` need to be updated when have no more entities to load. How can achieve this?

Today the component continuously call the api (incrementing the page) but with no data...

```js

// mixin
import InfinityModel from 'ember-infinity/lib/infinity-model';
import { computed } from '@ember/object';

export default InfinityModel.extend({
  _canLoadMore: computed('content', 'perPage', function() {
    console.log('I am just called once!');
    
    let results = this.get('content').length;

    return results > 0 && results == this.get('perPage');
  }).readOnly()
})

// route...
import Route from './list-route';
import InfinityLoader from '../mixins/infinity';

export default Route.extend({
  model(params) {
    params = this._super(...arguments); // setting startingPage, perPage n others.

    return this.infinityModel('opinion', params, InfinityLoader);
  },

  afterInfinityModel(posts) {
    console.log('I am not fired :/');
  }
});
```
 
My ideia is load all entities but when a page was returned with data insufficient (lower than a full page) I get false, and then the message was showed up.

Any help is appreciated!

### PR DESCRIPTION
**Pull request by snewcomer**
Close #285

### CODE REVIEW

File: addon/services/infinity.js [hunk 1]
```diff
@@ -296,6 +276,24 @@ export default Service.extend({
...
+    if (!this.afterInfinityModel || typeof this.afterInfinityModel !== 'function') {
+      return newObjects;
+    }
+
+    let result = this.afterInfinityModel(newObjects, infinityModel);
+    if (result) {
+      return result;
+    }
+    return newObjects;
+  },
```
Comment (snewcomer) [inline addon/services/infinity.js]: We are moving towards a service based approach; however, this is to fix backwards compat support.

**Comment by johncalvinyoung** [comment]
Looks good. Yes please! (trying to move an existing app to 1.0.0-beta.4 because I need the service for a new feature).

**Comment by johncalvinyoung** [comment]
For writing to `infinityModel.canLoadMore` for cursor-based pagination as in the example updated in #286, this line will have to change: https://github.com/ember-infinity/ember-infinity/pull/286/files#diff-5d97a8d6defce9ac36b641bb11772ad5R135

As currently written, this defines a circular dependency, and attempting to write results in a stack overflow.

**Comment by brunoocasali** [comment]
@snewcomer I've made the change proposed by @johncalvinyoung and it worked! (more information in original issue) :D

**Comment by brunoocasali** [comment]
@snewcomer I think the build is broken because the Travis doesn't work as expected, maybe run the build again, will fix it? #289 is green :)

**Pull request merged**